### ISSUE TITLE
Update Muse 4 in the debian repositories.

### ISSUE THREAD

**Issue by Gabrielxd195** [muse-sequencer/muse#1025]
I would like to know when will muse 4.0 be released for the debian repositories? since I need specifically the debian version for a personal project. I have no problem using the appimage version (which by the way I love how it is constantly updated) it works very well for me. I have seen that for now muse 4 is only in the arch, kaos and flathub repositories. I have tried to compile it from source code but apart from not compiling, the times I have tried it gives me errors. Greetings

**Comment by terminator356** [comment]
That is kind of out of our hands. I have reached out to them and filed an issue a few times but nothing happened.
They said there were a couple of problems with the app that made them reluctant to update it.
I pointed out that the specific problems had long ago been fixed.
I said I have no experience making a deb package but that I would try, but I gave up almost immediately
 because I don't know anything about it.
So either someone must step up and make a deb package or they should do it.
